Cargo.toml

```diff
@@ -19,6 +19,7 @@ scale-info = { version = "2.11.3", default-features = false }
 [dev-dependencies]
 ink_e2e = "5.0.0"
 test_helpers = { path = "src/test_helpers", default-features = false }
+scale-info = "2.11.3"
```

Comment:
what's the reason for adding this `dev-dependencies`?

Reply:
rust-analyzer gets fussy without it for me locally, my bad will remove